jalali_test/jalali_test/settings.py

```diff
@@ -83,6 +83,23 @@
 }
 
 
+# Running tests based on selected database 
+
+if 'test' in sys.argv:
+    entry_list = ["mysql", "MySQL", "MYSQL"]
+    entry = input("Do you want to test with SQLite or MySQL ? ")
+    if entry in entry_list:    
+        DATABASES={
+                "default": {
+                    "ENGINE": "django.db.backends.mysql",
+                    "NAME": "django_jalali_db",
+                    "USER": "django_jalali_user",
+                    "HOST": "127.0.0.1",
+                    "PASSWORD": "password",
+                    "PORT": "3306",
+                },
+            }
+    
```

Comment:
I think this does not work on an automated test pipeline in Github action. You can see all tests failed.